src/test/java/teammates/test/cases/datatransfer/AdminEmailAttributesTest.java

```diff
@@ -0,0 +1,113 @@
+package teammates.test.cases.datatransfer;
+
+import static teammates.common.util.Assumption.assertEquals;
+import static teammates.common.util.Assumption.assertFalse;
+import static teammates.common.util.Assumption.assertTrue;
+import static teammates.common.util.FieldValidator.getPopulatedErrorMessage;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.google.appengine.api.datastore.Text;
+
+import teammates.common.datatransfer.attributes.AdminEmailAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.FieldValidator;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.StringHelper;
+import teammates.storage.entity.AdminEmail;
+import teammates.test.driver.StringHelperExtension;
+
+/**
+ * SUT: {@link AdminEmailAttributes}.
+ */
+public class AdminEmailAttributesTest {
```

Comment:
This should extend `BaseAttributesTest`. That way, none of the above static imports are necessary.